packages/squarev2-integration/src/squarev2-payment-processor.ts

```diff
@@ -106,21 +107,8 @@ export default class SquareV2PaymentProcessor {
         return result.token;
     }
 
-    async verifyBuyer(token: string): Promise<string> {
-        const { getCheckoutOrThrow, getBillingAddressOrThrow } =
-            this._paymentIntegrationService.getState();
-        const { outstandingBalance, cart } = getCheckoutOrThrow();
-
-        const details: ChargeVerifyBuyerDetails = {
-            amount: outstandingBalance.toString(),
-            billingContact: this._mapToSquareBillingContact(getBillingAddressOrThrow()),
-            currencyCode: cart.currency.code,
-            intent: 'CHARGE',
-        };
-
-        const response = await this._getPayments().verifyBuyer(token, details);
-
-        return response ? response.token : '';
+    async verifyBuyer(token: string, intent: 'CHARGE' | 'STORE'): Promise<string> {
```

Comment:
You should create `enum` type for intent. Take a look here https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/payment/strategies/square/square-form.ts#L29